src/test/java/roomescape/auth/domain/MemberTest.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.auth.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.reservation.domain.Name;
+
+class MemberTest {
+
+  @DisplayName("어드민 권한 여부를 반환한다 - 참.")
+  @Test
+  void isAdmin() {
+    // given
+    Member member = new Member(
+        1L,
+        new Name("네오"),
+        "neo@example.com",
+        "password123",
+        new Role("ADMIN")
+    );
+    // when
+    boolean isAdmin = member.isAdmin();
+    // then
+    Assertions.assertThat(isAdmin).isTrue();
```

Comment:
```suggestion
   assertThat(isAdmin).isTrue();
```
기존 테스트처럼 `Assertions` 부분은 빼도 괜찮을 것 같아요.

Reply:
일관성 리뷰의 개수가 제 꼼꼼하지 못함을 드러내고 있군요..